extensions/stackdriver/common/context.cc

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2019 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "extensions/stackdriver/common/context.h"
+#include "extensions/stackdriver/common/constants.h"
+
+namespace Extensions {
+namespace Stackdriver {
+namespace Common {
+
+// Find value of the given key from the metadata proto map and copy it into the
+// field. If the given key cannot be found, field remains unset.
+void FillNodeMetadataField(
```

Comment:
Return a bool?

Reply:
The function has been removed and the new code returns JSON parsing status.